app/javascript/src/components/Dashboard/Contacts/Pane/CreateOrEdit.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import { Pane, Typography } from "neetoui";
+
+import Form from "./Form";
+
+const CreateOrEditPane = ({
```

Comment:
Please explain to me your thought process and advantages that you saw while naming the file as `CreateOrEdit.jsx` and naming the component as `CreateOrEditPane`.